perfzero/lib/perfzero/perfzero_config.py

```diff
@@ -97,6 +97,23 @@ def add_setup_parser_arguments(parser):
       --build-arg arg0 --build-arg arg1=value1 --build-arg "arg2=value with space" --build-arg arg3=300
       '''
       )
+  parser.add_argument(
+      '--docker_pull',
+      action='store_true',
+      dest='docker_pull',
+      help='''Set to check remote repositories for updated images. This is the
+      default. Use --no-docker_pull for the opposite.
+      '''
+      )
+  parser.add_argument(
+      '--no-docker_pull',
```

Comment:
What do you think about this?  @sganeshb   I think this is much better. I over did it the first time. It was getting late.

That is the downside to argparse you cannot do a boolean like you would with absl.  It works fine if you want to only have true and default false.  Here is another option that gets a very similar result (maybe the same really) but with less lines.  

This creates only a `--no-docker_pull` and **no** `docker_pull` (which makes sense as it is True by default).  `--no-docker_pull` sets `docker_pull` True by default and then flips to false when `--no-docker_pull` is used.  

Note:  that if you pass --docker_pull on the commandline it will fail because it is not a flag, which I think is fine.

```python
  parser.add_argument(
      '--no-docker_pull',
      action='store_false',
      default=True,
      dest='docker_pull',
      help='''Set to skip checking remote repositories for updated images. This
      is useful if the base image has been created locally.
      '''
      )

```

Reply:
Thanks - that looks better. Should the default be False to preserve old behavior?